version.ml

```diff
@@ -0,0 +1 @@
+let version = "1.0"
```

Comment:
What is this for?

Reply:
The CLI tools display a version number when listing help. I believe we have now decided that this is version 1.0 going forward.